.cirrus.yml

```diff
@@ -568,6 +569,18 @@ remote_system_test_task:
         PODBIN_NAME: remote
 
 
+container_system_test_task:
+    <<: *local_system_test_task
+    alias: container_system_test
+    skip: *branch
```

Comment:
Trivial low-priority question: is this `*branch` deliberate? In looking at the Cirrus graph, I notice that this is the only one of the right-hand boxes (system tests) that has it set; all the others -- rootless, local, remote -- skip on CI:DOCS but not on BRANCH. I don't see a strong reason to go either way, I'm just asking because it's a curious inconsistency.
![cirrus-map-pr8727](https://user-images.githubusercontent.com/265639/102248822-dd279800-3ebe-11eb-934c-f6dedaf29a50.png)

Reply:
Is it just me, or is the graph beginning to look like a... um... cetacean?